docs/adasum_user_guide.rst

```diff
@@ -63,19 +63,17 @@ Setting up the environment
 
 Below are the requirements for running Horovod with AdaSum:
 
--   cuda-10.0 and cudnn-7.6.4
+-   cuda >= 6.0
 
--   OpenMPI- 4.0
+-   OpenMPI >= 3.0
 
--   NCCL – 2.4
+-   NCCL >= 2.0
 
--   Python 3.7.5
+-   Pytorch >= 1.2.0 OR
 
--   Pytorch-1.2.0 OR
+-   Tensorflow >= 1.14.0
```

Comment:
@ashbhandare, does Adasum support TensorFlow 2?  Last I checked it was only for TensorFlow 1.

Reply:
That is correct, added the requirement.